website_snippet_carousel_product/static/src/scss/s_product_carousel.scss

```diff
@@ -0,0 +1,31 @@
+// Copyright 2020 Tecnativa - Alexandre Díaz
+// License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl).
+.s_product_carousel {
+    .oe_product {
+        border-width: 0;
+
+        .oe_product_image {
+            position: unset;
+            padding: 15px;
+
+            img {
+                position: unset;
+            }
+        }
+    }
+
+    .carousel-control-prev {
+        width: 5%;
+        left: -5%;
```

Comment:
You can use standard Bootstrap markup to solve these problems in a way that is more maintainable and readable IMHO, as explained [here](https://stackoverflow.com/a/59377509), and make it look pretty much the same:

![layout](https://user-images.githubusercontent.com/973709/84122390-47eba880-aa30-11ea-98f6-65aae919ac51.png)

Reply:
I prefer user "d-flex" that works better with small screens